multicast/send.py

```
@@ -354,7 +354,11 @@ def doStep(self, *args, **kwargs):
 			# Read from stdin in chunks
 			while True:
 				try:
-					chunk = sys.stdin.read(1316)  # Read 1316 bytes at a time - matches read size
+					# Read configured amount of bytes at a time - matches read size by default
+					# skipcq: PYL-W0212
+					chunk = sys.stdin.read(
+						multicast._MCAST_DEFAULT_BUFFER_SIZE  # skipcq: PYL-W0212 - module ok
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)